### PR TITLE
Config examples

### PR DESCRIPTION
**Pull request by larsespenssb**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ repos:
         types: [python]
       - id: ruff
         name: ruff
-        entry: ruff check --fix --exit-non-zero-on-fix
+        entry: ruff --fix --exit-non-zero-on-fix
```
Comment (arneso-ssb) [inline .pre-commit-config.yaml:54]: Keep the check-option. Otherwise you get warnings.

File: config_examples/config_example.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    from config_examples.ConfigReader import ConfigReader
+
+# %%
+config = ConfigReader()
+
+print(config.config["paths"]["inndata_dir"])
+
+
+# %%
+sti = config.klargjort_dir()
```
Comment (arneso-ssb) [inline config_examples/config_example.py:27]: Use English variable name, for example "dir" or "directory".

File: parquet/dataset.md [hunk 3]
```diff
@@ -84,10 +84,3 @@ This database structure enables statistical analysis related to customer orders,
 such as calculating total revenue, average order value, popular products, and sales
 by category. It can support queries and analysis to gain insights into customer
 behavior and product performance.
-
```
Comment (arneso-ssb) [inline parquet/dataset.md:87]: Do not remove this section.

File: parquet/parquet_metadata.py
Comment (arneso-ssb) [inline parquet/parquet_metadata.py:1]: Do not remove this file.

File: parquet/sqltest.py
Comment (arneso-ssb) [inline parquet/sqltest.py:1]: Do not remove this file.

File: poetry.lock
Comment (arneso-ssb) [inline poetry.lock:1]: Do not delete this file. Recreate it with poetry update.

**Review by arneso-ssb — CHANGES_REQUESTED** [review]
In addition to the comments in the files, run: `poetry run pre-commit run --all-files` to fix trailing whitespace, missing endline, black formatting, isort etc.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (larsespenssb) [reply .pre-commit-config.yaml]: fixed in latest commit

**Review by arneso-ssb — APPROVED** [review]

**Pull request merged**